### PR TITLE
change yq binary to v4

### PR DESCRIPTION
**Pull request by vgupta-mickey**
## Purpose
[OBSDEF-nnnn](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-nnnn)
changed the makefile to start using the latest yq binary v4 which is the official release of yq.

## PR checklist
- [x ] make test passed
- [x ] make build passed
- [ ] helm install <chart> passed
- [x ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

### CODE REVIEW

**Review by eddiepavkovic — DISMISSED** [review]
what testing was done?  did we do `make build`, `make package` and `make release`

did we test the new plugin to ensure its the same as before?

File: Makefile [hunk 1]
```diff
@@ -82,8 +81,8 @@ decksver:
 	echo "Found it"
```
Comment (eddiepavkovic) [inline Makefile:81]: we should run a `yq --version` and check its v4 before continuing.

File: Makefile [hunk 2]
```diff
@@ -69,7 +68,7 @@ dep:
  	fi
 	export PATH=$PATH:/tmp
 	sudo pip install yamllint=="${YAMLLINT_VERSION}"
-	sudo pip install yq=="${YQ_VERSION}"
+	sudo snap install yq
```
Comment (tmarcu) [inline Makefile:71]: This will always install latest and may break us in the future if they update again, right?

Comment (vgupta-mickey) [reply Makefile]: Already taken care. Thanks.

**Comment by vgupta-mickey** [comment]
> what testing was done? did we do `make build`, `make package` and `make release`
> 
> did we test the new plugin to ensure its the same as before?

All tested:
make build
make test
make release
make package

**Review by eddiepavkovic — APPROVED** [review]
nice changes.  thanks.

**Review by tmarcu — APPROVED** [review]

**Pull request merged**